### ISSUE TITLE
i18n rule reports error when there is a HTML tag in a tag using i18n attribute

### ISSUE THREAD

**Issue by dixy** [angular-eslint/angular-eslint#327]
With the `@angular-eslint/template/i18n` configured like this:
```js
{
  checkId: true,
  checkText: false,
  checkAttributes: false,
}
```

An error is reported when there are HTML tags in texts: _Missing custom message identifier._
In the example below, an error is reported on the `<i>`:
```html
<p i18n="@@customId">Lorem ipsum <i>dolor</i> sit amet.</p>
```

Unless I am mistaken, I don't think it is the intended behavior?

**Comment by janousek** [comment]
How to fix this?
Change this line:
https://github.com/angular-eslint/angular-eslint/blob/eeb49d491bb92e470f505a699ef3e12963ff351b/packages/eslint-plugin-template/src/rules/i18n.ts#L221
To this:
```
if (node.i18n && !node.parent?.i18n) {
```

And this line:
https://github.com/angular-eslint/angular-eslint/blob/eeb49d491bb92e470f505a699ef3e12963ff351b/packages/eslint-plugin-template/src/rules/i18n.ts#L233
To this:
```
else if (!node.i18n)
```

**Comment by rafaelss95** [comment]
@janousek would you mind to create a PR with this proposed fix?
